Pages/HostSelectorPage.xaml.cpp

```diff
@@ -56,14 +56,14 @@ void moonlight_xbox_dx::HostSelectorPage::NewHostButton_Click(Platform::Object^
 void moonlight_xbox_dx::HostSelectorPage::OnNewHostDialogPrimaryClick(Windows::UI::Xaml::Controls::ContentDialog^ sender, Windows::UI::Xaml::Controls::ContentDialogButtonClickEventArgs^ args)
 {
 	sender->IsPrimaryButtonEnabled = false;
-	Platform::String^ hostname = dialogHostnameTextBox->Text;
+	Platform::String^ ipAddress = dialogHostnameTextBox->Text;
 	auto def = args->GetDeferral();
-	Concurrency::create_task([def, hostname, this, args, sender]() {
-		bool status = state->AddHost(hostname);
+	Concurrency::create_task([def, ipAddress, this, args, sender]() {
```

Comment:
Replacing the bot comment here, users expect to input the port number too here and not hust the IP Address, valid input combination can be:
-192.168.1.1
- 192.168.1.1:47989
- DESKTOP-AAAA:47989 (via local hostname resolution)
- servername.mymoonlighthostingservice.com:47989 (some niche providers do expose these hosts over the internet using DNS names)

We can split IP and port directly here instead of keeping lastHostname